plugins/arDominionPlugin/css/less/buttons.less

```diff
@@ -170,6 +170,10 @@ button.clipboard-wide {
   }
 }
 
+button.clipboard.clipboard-repository-browse {
```

Comment:
You could have place this inside the `button.clipboard` section bellow using `&.clipboard-repository-browse`. The transformation to CSS will be the same but it makes a nicer LESS with the nested rules.